<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSInitConditionsDialog</class>
 <widget class="QDialog" name="DSInitConditionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>271</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Начальные условия</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="harmonicLabel">
     <property name="text">
      <string>Гармоника</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="activatorCoeffLabel">
     <property name="text">
      <string>Коэфф. активатора</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="harmonicCombo0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="activatorEdit0">
     <property name="text">
      <string>1.0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QLineEdit" name="inhibitorEdit0">
     <property name="text">
      <string>1.0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="harmonicCombo1"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="activatorEdit1">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QLineEdit" name="inhibitorEdit1">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="harmonicCombo2"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="activatorEdit2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QLineEdit" name="inhibitorEdit2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="harmonicCombo3"/>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="activatorEdit3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QLineEdit" name="inhibitorEdit3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="harmonicCombo4"/>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="activatorEdit4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QLineEdit" name="inhibitorEdit4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="harmonicCombo5"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="activatorEdit5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QLineEdit" name="inhibitorEdit5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="6">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>372</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="inhibitorCoeffLabel">
     <property name="text">
      <string>Коэфф. ингибитора</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="removeHarmonicButton1">
     <property name="text">
      <string>Удалить</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="removeHarmonicButton2">
     <property name="text">
      <string>Удалить</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="removeHarmonicButton3">
     <property name="text">
      <string>Удалить</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="removeHarmonicButton4">
     <property name="text">
      <string>Удалить</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="removeHarmonicButton5">
     <property name="text">
      <string>Удалить</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DSInitConditionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DSInitConditionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
